components/atom/progressBar/src/ProgressBarLine/settings.scss

```
@@ -1,11 +1,17 @@
 $bg-progress-bar: $c-gray-lightest !default;
+$bg-progress-bar--success: $bg-progress-bar !default;
+$bg-progress-bar--loading: $bg-progress-bar !default;
+$bg-progress-bar--error: $bg-progress-bar !default;
 $bc-progress-bar: $c-gray-light !default;
 $c-progress-bar-fill: $c-primary !default;
 
 $trsdu-progress-bar: 500ms !default;
 $trstf-progress-bar: linear !default;
 
 $bd-line-progress-container: $bdw-s solid $bc-progress-bar !default;
+$bd-line-progress-container--success: $bd-line-progress-container !default;
+$bd-line-progress-container--loading: $bd-line-progress-container !default;
+$bd-line-progress-container--error: $bd-line-progress-container !default;
```

IMHO, according to our naming convention, variables like this:

```suggestion
$bd-line-progress-container
```

should be named 
```suggestion
$bd-progress-bar-container-line
```

the first part after the emmet bit (`bd`), should reference the component, `progress-bar` in this case.

---

After discussing with Jordi, we're going to leave it as it is now to avoid backward compatibility issues. We'll talk this with Andres at some moment.